### DISCUSSION TITLE
Is 0.6.5 an official release?

### DISCUSSION THREAD

**Discussion by rGunti** [alirezanet/Husky.Net#112] [Q&A] [answered]
I have received an alert from our Renovate bot that husky is getting an update to 0.6.5, as evident here on NuGet (https://www.nuget.org/packages/Husky/0.6.5). However I don't see 0.6.5 on the Releases page of this repo (https://github.com/alirezanet/Husky.Net/releases) so I hesitate to approve the update into our code base.
Is 0.6.5 a real, official release or did something bad happen to the owners credentials?

**Comment by alirezanet** [comment, marked as answer]
Hi @rGunti,
Yes, it is, I forgot to create a release for this one because I haven't received any confirmation whether it solves the bug in #107.
feel free to update it and please let me know if you have any problems with this version. I'll create a release soon,
